source/transactions/transactions.rst

```diff
@@ -492,19 +561,18 @@ session is in the "starting transaction" state, meaning, no operations
 have been performed on this transaction, drivers MUST NOT run the
 abortTransaction command.
 
-abortTransaction is a retryable write command. Drivers MUST retry once
+abortTransaction is a retryable write command. Drivers MUST retry
```

Comment:
Is there a reason we wouldn't? My understanding was that commit/abort transaction should be treated as normal retryable writes.

Reply:
It's similar to closing a cursor.  The database will eventually clean up any uncommitted transactions, so both should be done as "best effort".  That is my understanding.  I think there is a note to that effect just below?

Reply:
It's conceptually similar because both are resource cleanup attempts, but in practice, failure to abort a transaction can leave a database/collection in an unusable state for up to a minute, so in my opinion it's imperative to do the best we can here. The note below still applies if we retry multiple times. If the abort eventually fails due to a server selection failure or a non-retryable error (e.g. after a transient network issue, if we retry commit/abort on a different mongos, it'll fail with a `NoSuchTransaction` error), drivers will swallow that error as they already do now.